lib/fastlane/plugin/ddg_apple_automation/helper/git_helper.rb

```diff
@@ -104,6 +135,9 @@ def self.latest_release(repo_name, prerelease, platform, github_token)
           # If `prerelease` is false, then ensure that the release is public.
           matching_release = releases.find do |release|
             matches_platform = platform.nil? || release.tag_name.end_with?("+#{platform}")
+            if allow_drafts
+              matches_platform ||= release.name.end_with?("+#{platform}")
```

Comment:
Drafts don’t need to have a tag assigned, so we check the name of the release (in our case, names are equal to tags anyway).